cmd/wallet-sdk-gomobile/api/diddocresolution.go

```diff
@@ -34,3 +34,22 @@ func (d *DIDDocResolution) ID() (string, error) {
 
 	return didDocResolutionParsed.DIDDocument.ID, nil
 }
+
+// AssertionMethodKeyID returns the id of key that used for assertion verification.
+func (d *DIDDocResolution) AssertionMethodKeyID() (string, error) {
```

Comment:
Perfect place to put this method. This will make unifying the keyID approach in both OpenID4CI and OpenID4VP much easier.